### ISSUE TITLE
map-markers-overlay endpoint documentation isn't up to date

### ISSUE THREAD

**Issue by bobular** [VEuPathDB/EdaDataService#191]
https://veupathdb.github.io/EdaDataService/api.html#resources:/apps/pass/visualizations/map-markers-overlay:post

Maybe I'm looking at the wrong thing, but in the response specs, `mapMarkers.data[].facetVariableDetails` is an array, but it needs to be `mapMarkers.data[].geoAggregateVariableDetails` and it needs to be a non-array.

**Comment by bobular** [comment]
Also, in `sampleSizeTable` see https://veupathdb.github.io/EdaDataService/api.html#type:lib.SampleSizeTable 
`xVariableDetails` is spec'ed to be an array, but there can only be one x variable, I think.

There are already responses where xVariableDetails.value is an array, and that's OK, I think (but the spec says not)

However, sometimes xVariableDetails.value is a non-array (when there is only a single value) - is this R trying to be helpful again?

TLDR; please review sampleSIzeTable xVariableDetails API specs
